config/rbac/key-management-role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: storageos-key-management
```

Comment:
`storageos:key-management`

Reply:
Controller runtime doesn't support names with `:` that's why i changed every to `-`. The error during generation is:
```
Error: unable to parse option "rbac:roleName=storageos:api-manager": [expected comma, got ":" (at <input>:1:19) extra arguments provided: "api-manager" (at <input>:1:20)]
```